web/src/components/SpanDetail/components/AssetionsSpanComponent.tsx

```diff
@@ -0,0 +1,37 @@
+import {Typography} from 'antd';
+import {ISpan} from 'types/Span.types';
+import SkeletonTable from 'components/SkeletonTable';
+import AssertionsResultTable from 'components/AssertionsTable/AssertionsTable';
+import * as S from 'components/SpanDetail/SpanDetail.styled';
+
+interface IProps {
+  span?: ISpan;
+  testId?: string;
+  resultId?: string;
+  assertionsResultList: any[];
+}
+
+export const AssetionsSpanComponent = ({assertionsResultList, testId, span, resultId}: IProps): JSX.Element => (
+  <SkeletonTable loading={!span}>
+    {assertionsResultList.length ? (
+      assertionsResultList
+        .sort((a, b) => (a.assertion.assertionId > b.assertion.assertionId ? -1 : 1))
+        .map(({assertion, assertionResultList}, index) => (
+          <AssertionsResultTable
+            key={assertion.assertionId}
+            assertionResults={assertionResultList}
+            sort={index + 1}
+            assertion={assertion}
+            span={span!}
+            resultId={resultId!}
+            testId={testId!}
+          />
+        ))
+    ) : (
+      <S.DetailsEmptyStateContainer>
```

Comment:
I think the empty data style also changed
![Screen Shot 2022-05-06 at 8 10 51 a m](https://user-images.githubusercontent.com/11051031/167137966-5bf35e08-44c7-4913-81a7-ac93ef6437ff.png)